src/index.html

```diff
@@ -9,6 +9,14 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <main>
+      <div class="watch">
+        <div class="watch__hand-big"></div>
+
+        <div class="watch__hand-small"></div>
+
+        <div class="watch__dot"></div>
```

Comment:
```suggestion
        <div class="watch__hand-big"></div>
        <div class="watch__hand-small"></div>
        <div class="watch__dot"></div>
```

No need to add empty lines between block elements